addons/web/static/tests/core/position_hook_tests.js

```diff
@@ -150,6 +150,42 @@ QUnit.test("has no effect when component is destroyed", async (assert) => {
     );
 });
 
+QUnit.test("is positioned relative to its containing block", async (assert) => {
+    assert.expect(4);
```

Comment:
not needed, we try to only specify `assert.expect(x)` if the number of assertion could be variable (like asserting inside a patched function or the makeView util mockRPC param.